apps/web/components/NameSignupForm/NameSignupForm.tsx

```diff
@@ -56,6 +59,14 @@ export const NameSignupForm = ({ namespace, slice }: NameSignupFormProps) => {
     return errors
   }
 
+  const [
+    submit,
```

Comment:
This comment should be treated as a feedback (not requiring code-changes, but it would improve it)

Same goes for the other file, a bit nit-picky; but improves code readability.
It would improve the code if this would be explicitly named, i.e. if this would be e.g. `subscribeToMailchimp`.
`submit` is too broad, since this is explicitly calling a mutation with side-effects. This component could e.g. have more forms/mutations.